Dockerfile

```diff
@@ -105,3 +111,32 @@ EXPOSE 8888
 ENTRYPOINT ["/home/rapids/entrypoint.sh"]
 
 CMD [ "sh", "-c", "jupyter-lab --notebook-dir=/home/rapids/notebooks --ip=0.0.0.0 --no-browser --NotebookApp.token='' --NotebookApp.allow_origin='*' --NotebookApp.base_url=\"${NB_PREFIX:-/}\"" ]
+
+# Labels for NVIDIA AI Workbench
+LABEL com.nvidia.workbench.application.jupyterlab.class="webapp"
+LABEL com.nvidia.workbench.application.jupyterlab.health-check-cmd="[ \\$(echo url=\\$(jupyter lab list | head -n 2 | tail -n 1 | cut -f1 -d' ' | grep -v 'Currently' | sed \"s@/?@/lab?@g\") | curl -o /dev/null -s -w '%{http_code}' --config -) == '200' ]"
+LABEL com.nvidia.workbench.application.jupyterlab.start-cmd="jupyter lab --allow-root --port 8888 --ip 0.0.0.0 --no-browser --NotebookApp.base_url=\\\$PROXY_PREFIX --NotebookApp.default_url=/lab --NotebookApp.allow_origin='*'"
+LABEL com.nvidia.workbench.application.jupyterlab.webapp.port="8888"
+LABEL com.nvidia.workbench.application.jupyterlab.stop-cmd="jupyter lab stop 8888"
+LABEL com.nvidia.workbench.application.jupyterlab.type="jupyterlab"
+LABEL com.nvidia.workbench.application.jupyterlab.webapp.url-cmd="jupyter lab list | head -n 2 | tail -n 1 | cut -f1 -d' ' | grep -v 'Currently'"
+LABEL com.nvidia.workbench.application.jupyterlab.webapp.autolaunch="true"
+LABEL com.nvidia.workbench.cuda-version="$CUDA_VER"
+LABEL com.nvidia.workbench.description="RAPIDS with CUDA ${CUDA_VER}"
+LABEL com.nvidia.workbench.entrypoint-script="/home/rapids/entrypoint.sh"
+LABEL com.nvidia.workbench.labels="cuda${CUDA_VER}"
+LABEL com.nvidia.workbench.name="RAPIDS with CUDA ${CUDA_VER}"
+LABEL com.nvidia.workbench.os="linux"
+LABEL com.nvidia.workbench.os-distro="$LINUX_DISTRO"
+LABEL com.nvidia.workbench.os-distro-release="$LINUX_DISTRO_VER"
+LABEL com.nvidia.workbench.package-manager.apt.binary="/usr/bin/apt"
+LABEL com.nvidia.workbench.package-manager.apt.installed-packages="curl git git-lfs vim"
```

Comment:
these packages are not in this container. Either remove them from this label, or install.